README.md

```diff
@@ -105,6 +105,17 @@ Begin Transaction with Activation token:
 AuthenticationResponse beginResponse = idxAuthenticationWrapper.begin("activation-token");
 ```
 
+Begin Transaction with Device Context:
+
+```java
+RequestContext requestContext = new RequestContext();
+requestContext.addXDeviceTokenHeader("test_x_device_token");
+requestContext.addXOktaUserAgentExtendedHeader("test_x_okta_user_agent_extended");
+requestContext.addXForwardedForHeader("test_x_forwarded_for");
+
+AuthenticationResponse beginResponse = idxAuthenticationWrapper.begin(requestContext);
+```
+
```

Comment:
We will need to provide guidance on the Device Token, what our recommendations are, how its used, etc.

Reply:
Sure, we can include a link to developer docs here when it is ready.